optimum/commands/onnxruntime/optimize.py

```diff
@@ -0,0 +1,71 @@
+from argparse import ArgumentParser
+from pathlib import Path
+
+from ...onnxruntime.configuration import AutoOptimizationConfig
+from ...onnxruntime.optimization import ORTOptimizer
+
+
+def parse_args_onnxruntime_optimize(parser):
+    required_group = parser.add_argument_group("Required arguments")
+    required_group.add_argument(
+        "--onnx_model",
+        type=Path,
+        required=True,
+        help="Path to the repository where the ONNX models to optimize are located.",
+    )
+
+    optional_group = parser.add_argument_group("Optional arguments")
+    optional_group.add_argument(
+        "-o",
+        "--output",
+        type=Path,
+        help="Path to the directory where to store generated ONNX model. (defaults to --onnx_model value).",
+    )
+
```

Comment:
Being able to provide predefined optimization configs is great.
Could we also add the possibility to provide a path where an `ORTConfig` is stored?

Reply:
I can add this yes.

Reply:
I think it would allow for more custom usage!

Reply:
@michaelbenayoun Do you mean `ORTConfig` or `OptimizationConfig`?

Reply:
Done!